Makefile

```diff
@@ -272,11 +267,27 @@ db-migrate-upgrade:
 db-schema: ## Run db-schema varchar tests
 	cd $(MAKEFILE_ROOT)/backend-shared && go run ./hack/db-schema-sync-check
 
+### --- K C P --- ###
+
 gen-kcp-api-backend-shared: ## Runs utilities/generate-kcp-api-backend-shared.sh to generate kcp api resource schema and export
 	cd $(MAKEFILE_ROOT)/utilities && ./generate-kcp-api-backend-shared.sh
 
 gen-kcp-api-appstudio-shared: ## Runs utilities/generate-kcp-api-appstudio-shared.sh to generate kcp api resource schema and export
 	cd $(MAKEFILE_ROOT)/utilities && ./generate-kcp-api-appstudio-shared.sh
 
 kcp-test-local-e2e: ## Initiates a ckcp within openshift cluster and runs e2e test
-	cd $(MAKEFILE_ROOT)/kcp && ./ckcp/setup-ckcp-on-openshift.sh
\ No newline at end of file
+	cd $(MAKEFILE_ROOT)/kcp && ./ckcp/setup-ckcp-on-openshift.sh
+
+gen-kcp-api-all: gen-kcp-api-appstudio-shared gen-kcp-api-backend-shared ## Creates all the KCP API Resources for all comfig/crds
+
+apply-kcp-api-all: ## Apply all APIExport to the cluster
+	$(MAKEFILE_ROOT)/utilities/create-apiexports.sh
+
```

Comment:
When I run this, I get
```
make setup-e2e-kcp-virtual-workspace
```
I get:
```
./kcp/utils.sh: line 309: syntax error near unexpected token `}'
```
looks like there's a missing `fi` in utils.sh